packages/@react-aria/checkbox/docs/useCheckboxGroup.mdx

```diff
@@ -122,16 +122,23 @@ function Checkbox(props) {
   let ref = React.useRef();
   let {inputProps} = useCheckboxGroupItem(props, state, ref);
 
+  let isDisabled = state.isDisabled || props.isDisabled;
+  let isSelected = state.isSelected(props.value);
+
   return (
-    <label style={{display: 'block'}}>
-      <input {...inputProps} />
+    <label
+      style={{
+        display: 'block',
+        color: (isDisabled && 'slategray') || (isSelected && 'royalblue'),
```

Comment:
Let's make sure these colors are accessible. We have some basic color vars to use in examples like this.
```suggestion
        color: (isDisabled && 'var(--gray)') || (isSelected && 'var(--blue)',
```